middleware/src/main/java/com/frontegg/sdk/middleware/context/FronteggContextHolder.java

```diff
@@ -0,0 +1,21 @@
+package com.frontegg.sdk.middleware.context;
+
+public class FronteggContextHolder {
```

Comment:
Doesn't spring already have a request context mechanism ? why do you need to reimplement it ?